posthog/api/user.py

```diff
@@ -40,6 +39,7 @@ class Meta:
         model = User
         fields = [
             "date_joined",
+            "id",
```

Comment:
Maybe worth merging #5971 first to make it super explicit what we're looking at here